config.md

```diff
@@ -350,17 +350,17 @@ Implementations MUST error out when invalid values are encountered and MUST gene
 Hooks allow for the configuration of custom actions related to the [lifecycle](runtime.md#lifecycle) of the container.
 
 * **`hooks`** (object, OPTIONAL) MAY contain any of the following properties:
-  * **`prestart`** (array, OPTIONAL) is an array of [pre-start hooks](#prestart).
-    Entries in the array contain the following properties:
-    * **`path`** (string, REQUIRED) with similar semantics to [IEEE Std 1003.1-2001 `execv`'s *path*][ieee-1003.1-2001-xsh-exec].
-      This specification extends the IEEE standard in that **`path`** MUST be absolute.
-    * **`args`** (array of strings, OPTIONAL) with the same semantics as [IEEE Std 1003.1-2001 `execv`'s *argv*][ieee-1003.1-2001-xsh-exec].
-    * **`env`** (array of strings, OPTIONAL) with the same semantics as [IEEE Std 1003.1-2001's `environ`][ieee-1003.1-2001-xbd-c8.1].
-    * **`timeout`** (int, OPTIONAL) is the number of seconds before aborting the hook.
-  * **`poststart`** (array, OPTIONAL) is an array of [post-start hooks](#poststart).
-    Entries in the array have the same schema as pre-start entries.
-  * **`poststop`** (array, OPTIONAL) is an array of [post-stop hooks](#poststop).
-    Entries in the array have the same schema as pre-start entries.
+    * **`prestart`** (array, OPTIONAL) is an array of [pre-start hooks](#prestart).
+        Entries in the array contain the following properties:
+        * **`path`** (string, REQUIRED) with similar semantics to [IEEE Std 1003.1-2001 `execv`'s *path*][ieee-1003.1-2001-xsh-exec].
+        This specification extends the IEEE standard in that **`path`** MUST be absolute.
```

Comment:
I think you want an additional two or four spaces of indent for this line so it is clearly part of the `path` entry in the source.

Other than that (and a `confif` → `config` typo in the commit message), 79afbeb looks good to me.